### PR TITLE
Fixes #18742: Remove plugin notice during upgrade

### PR DESCRIPTION
**Pull request by VinceMacBuche**
https://issues.rudder.io/issues/18742

### CODE REVIEW

**Comment by amousset** [comment]
Ok, for me, waiting for @Fdall approval too

**Review by fanf — APPROVED** [review]

**Comment by fanf** [comment]
It's ok with me too, and I would to be able to test it and know that we are ready for the final today

**Pull request merged**
